test/functional/framework.go

```diff
@@ -60,6 +60,11 @@ var outputLogFile = map[string]map[string]string{
 		k8sAuditLog:    "/var/log/infra.log",
 		ovnAuditLog:    "/var/log/infra.log",
 	},
+	logging.OutputTypeKafka: {
+		applicationLog: "/var/log/infra.log",
```

Comment:
I assume you want to send these to different log files to facilitate verification?

Reply:
@jcantrill i just followed - https://github.com/openshift/cluster-logging-operator/blob/ed0ac043934a256886613975f242163b16a9b697/test/functional/framework.go ... you are right ..these should different log files ...